src/idl.ts

```diff
@@ -986,15 +986,15 @@ export const IDL: Idl = {
       ],
       args: [
         {
-          name: 'disableMinting',
+          name: 'disable',
           type: 'bool',
         },
       ],
     },
   ],
   accounts: [
     {
-      name: 'controller',
+      name: 'Controller',
```

Comment:
this has to be in camel case iirc

Reply:
Yes not sure why it changed

Reply:
You must take these from the type.ts file, and the second half of the file (there are two parts)